src/Graph.ts

```diff
@@ -0,0 +1,174 @@
+/**
+ * Transform array to graph
+ */
+
+/**
+ * TYPES
+ */
+
+type node = {
+  x: Number;
+  y: Number;
+  value: Number;
+  isExit: Boolean;
+};
+
+type edge = {
+  from: string;
```

Comment:
even if a coordinate is indeed a string, I advise you to create a coordinates type for you to understand your function signatures better

Reply:
Coordinates type has been added and used with the [commit 82e57ce ](https://github.com/marmelab/kuba-integration/commit/82e57ce3a40dd7ed5da9fee6880050c75b8caa4b)